CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
-# Minimum CMake required
-cmake_minimum_required(VERSION 3.1)
+# Minimum CMake required.
+# FetchContent module requires 3.11.
+cmake_minimum_required(VERSION 3.11)
 include(cmake/Utils.cmake)
+include(FetchContent)
```

Comment:
Could we only include FetchContent when Build_Protobuf is needed?

Reply:
Good point, this will have better compatibility. Changed.